### ISSUE TITLE
checksum does not work on tar

### ISSUE THREAD

**Issue by byb121** [cancerit/archCompare#9]
```
$ cgpCompare -a cgpwgs-results-test112_run2/05abf7f2-15b1-11e7-afd4-957439c9df4b_vs_7b2cbc2e-15b5-11e7-b5b0-957439c9df4b/cgpwgs.result_WGS.tar.gz -b cgpwgs-results-test112_run1/05abf7f2-15b1-11e7-afd4-957439c9df4b_vs_7b2cbc2e-15b5-11e7-b5b0-957439c9df4b/cgpwgs.result_WGS.tar.gz -c checksum -j fileTypes.json 
Running analysis, please check log file for any errors 
detailed results in results.tsv file
input is an archive: cgpwgs-results-test112_run2/05abf7f2-15b1-11e7-afd4-957439c9df4b_vs_7b2cbc2e-15b5-11e7-b5b0-957439c9df4b/cgpwgs.result_WGS.tar.gz
input is an archive: cgpwgs-results-test112_run1/05abf7f2-15b1-11e7-afd4-957439c9df4b_vs_7b2cbc2e-15b5-11e7-b5b0-957439c9df4b/cgpwgs.result_WGS.tar.gz
Traceback (most recent call last):
  File "/nfs/users/nfs_y/yx2/projects/github/archCompare/env/bin/cgpCompare", line 11, in <module>
    load_entry_point('archCompare', 'console_scripts', 'cgpCompare')()
  File "/nfs/users/nfs_y/yx2/projects/github/archCompare/archCompare/compare_command.py", line 41, in main
    mycomp.run_comparison()
  File "/nfs/users/nfs_y/yx2/projects/github/archCompare/archCompare/compareArchive.py", line 320, in run_comparison
    results = self._get_sets_to_compare(dicta, dictb)
  File "/nfs/users/nfs_y/yx2/projects/github/archCompare/archCompare/compareArchive.py", line 249, in _get_sets_to_compare
    common_files = list(set(dictA.keys()) & set(dictB.keys()))
AttributeError: 'NoneType' object has no attribute 'keys'
```

**Comment by sb43** [comment]
Please email full paths and  I will test it

**Comment by sb43** [comment]
Corrected in release 1.1.0
